cmd/stolon-pgbouncer/main.go

```diff
@@ -300,7 +301,25 @@ func main() {
 			kingpin.Fatalf("failed to bind to address: %v", err)
 		}
 
-		server := pkgfailover.NewServer(logger, mustPgBouncer(pauserPgBouncerOptions))
+		bouncer := mustPgBouncer(pauserPgBouncerOptions)
+
+		// The pauser provides a safe API around pausing PgBouncer, where safe means the pause
+		// will eventually be removed. In the situation where our process was violently
+		// terminated, we may have broken this promise by never issuing a resume. By resuming
+		// whenever we restart, we rely on our supervisor system (systemd, kubernetes, etc) to
+		// bring our process back and immediately resume traffic, re-enabling queries.
+		//
+		// We may be mid-migration, but resuming too early is better than never resuming at
+		// all, which could lead to extended downtime.
+		logger.Log("event", "resuming_pgbouncer",
+			"msg", "resuming PgBouncer on start, in case we were terminated mid failover before")
```

Comment:
Looks like formatting is a bit off here.

Reply:
I think this is normal? I just broke on a new line and go will naturally lead with a tab

Reply:
Surely align msg with event if you’re doing that... or does gofmt not allow it?

Reply:
I'm not sure how you would align this- depending on what your tabstop is, it would align differently for different people right? I tend to never align things in a tab world for this reason

Reply:
Ah I forgot we’re in tab world here. Never mind then